doc/classes/RenderingServer.xml

```diff
@@ -1244,7 +1272,9 @@
 		<method name="fog_volume_create">
 			<return type="RID" />
 			<description>
-				Creates a new fog volume and allocates an RID.
+				Creates an fog volume and adds it to the RenderingServer. It can be accessed with the RID that is returned. This RID will be used in all [code]fog_volume_*[/code] RenderingServer functions.
```

Comment:
```suggestion
				Creates a fog volume and adds it to the RenderingServer. It can be accessed with the RID that is returned. This RID will be used in all [code]fog_volume_*[/code] RenderingServer functions.
```